test/ANNOTATIONS.yaml

```diff
@@ -1835,6 +1838,8 @@ spectralnorm:
     - qthreads updated to yield every ~100 uncontested sync var locks
   12/18/18:
     - brad version changed to serial implementation (#11904)
+  02/22/22:
```

Comment:
I'd go with the date we saw the impact for most configs. So probably the 18th -- https://chapel-lang.org/perf/chapcs/no-local/?startdate=2022/02/11&enddate=2022/02/24&configs=default,nolocal&graphs=spectralnormshootoutbenchmark

Reply:
Alright, updated. (I was confused by what you were saying about this in the meeting, but understand now)